docs/application/dotnet/guides/security/requesting-permissions.md

```diff
@@ -141,6 +141,91 @@ To check whether an application has permission to use a privilege, and to reques
 
+## Requesting Multiple Permissions
+
+This section describes how to check and request multiple privileges in a single API call.
+
+> **Note**
+>
+> Multiple privileges in a single API call are supported from Tizen 5.5.
+
+To check whether an application has permission to use a privilege, and to request permission if required:
+
+1. To verify whether an application has permission to use privileges, use `CheckPermissions()`:
+
+      ```csharp
+        string[] privileges = new [] {"http://tizen.org/privilege/account.read",
+                                      "http://tizen.org/privilege/alarm"};
+        CheckResult[] results = PrivacyPrivilegeManager.CheckPermissions(privileges).ToArray();
+      ```
+
+   The results of the call is stored in `CheckResult` array.
+
+2. React to the permissions check appropriately:
+
+      ```csharp
+        List<string> privilegesWithAskStatus = new List<string>();
+        try {
+            for (int iterator = 0; iterator &lt; results.Length; ++iterator)
+            {
+      ```
+   -   If the result value is `Allow`, the application is allowed to perform operations related to the privilege. For example, the application can enable additional UI elements or functionalities.
+         ```csharp
+                switch (results[iterator])
+                {
+                case CheckResult.Allow:
+                    // Privilege can be used
+                    break;
+         ```
+
+   -   If the result value is `Deny`, the application is not allowed to perform operations related to the privilege. Any attempt to use such functionality without the user's consent fails. Usually, this means that invoking any method that involves the privilege results in an error.
+         ```csharp
+                case CheckResult.Deny:
+                    // Privilege can't be used
+                    break;
+         ```
+
+   -   If the result value is `Ask`, the application must request permission from the user with the `RequestPermissions()` method, which displays a dialog box. When the user makes a decision, the answers are returned as
+  [Tizen.Security.RequestMultipleResponseEventArgs](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Security.RequestMultipleResponseEventArgs.html).
+
+         ```csharp
+                case CheckResult.Ask:
+                    // User permission request required
+                    privilegesWithAskStatus.Add(privileges[iterator]);
+                    break;
+                }
+         ```
+	  ```csharp
+            }
+            RequestMultipleResponseEventArgs request;
```

Comment:
If this code block is for whole "2. React to the permissions check appropriately:", then the alignment is correct. However, from a readers point of view, some confusions can arise.
How about giving an introductory line for the code block for  "RequestMultipleResponseEventArgs request;"?
Something related to the need to request user permissions, handle the user decisions. 

Please correct me if I am wrong.

Reply:
yes, it is.
There is no need to add more comments before or after 200 line.